openapi.yaml

```diff
@@ -0,0 +1,69 @@
+swagger: '2.0'
+info:
+  version: '1.0.0'
+  title: Fulcio
+schemes: [http, https]
+host: fulcio.sigstore.dev
+basePath: /v1
+consumes: [application/json]
+produces: [application/json]
+securityDefinitions:
+  Bearer:
+    type: apiKey
+    name: Authorization
+    in: header
+paths:
+  /submitcsr:
```

Comment:
wondering about making the externals:

POST `/signingCert` : create a cert, return content with a location header (with URL to CTL entry)